configuration/sitemaps.md

```diff
@@ -347,6 +348,24 @@ Webview url="http://www.openhab.org" height=5
 
 ![Presentation of the Webview element in BasicUI](images/sitemap_demo_webview.png)
 
+### Element Type 'Mapview'
+ 
+```perl
+Mapview [item=<itemname>] [label="<labelname>"] [icon="<iconname>"] [height=<heightvalue>]
```

Comment:
@Confectrician It seems that label and icon are not displayed in basic ui. Maybe this info should be added somewhere around here.

Reply:
Docs: https://github.com/openhab/openhab-docs/issues/640
ESH: https://github.com/eclipse/smarthome/issues/5030